Sources/DeepLearning/Operators/NN.swift

```diff
@@ -13,45 +13,61 @@
 // limitations under the License.
 
 #if !COMPILING_TENSORFLOW_MODULE
-import TensorFlow
+@_exported import TensorFlow
 #endif
 
-/// Returns the values of the specified tensor rounded to the nearest integer, element-wise.
-public func round<Scalar: BinaryFloatingPoint>(_ x: Tensor<Scalar>) -> Tensor<Scalar> {
-    return Raw.round(x)
-}
-
-/// Returns a tensor with the same shape and scalars as the specified tensor.
-@differentiable
-public func identity<Scalar>(_ x: Tensor<Scalar>) -> Tensor<Scalar> {
-    return x
-}
-
 //===------------------------------------------------------------------------------------------===//
 // Normalization
 //===------------------------------------------------------------------------------------------===//
 
-public extension Tensor where Scalar: TensorFlowFloatingPoint {
+public extension Tensor where Scalar: BinaryFloatingPoint {
```

Comment:
We should use `TensorFlowFloatingPoint` here for consistency with other core APIs. You can then drop the `where` clause in `@differentiable` and move the VJP to the same extension.